remodel/connection.py

```diff
@@ -1,5 +1,5 @@
 
-import rethinkdb as r
+import rethinkdb
```

Comment:
I would stick with the "traditional" conventions of using the `r` alias, unless you have other good reasons not to.